src/common/http-signature/verify-signature.middleware.ts

```diff
@@ -2,24 +2,27 @@ import { Injectable, NestMiddleware } from '@nestjs/common';
 import { NextFunction, Request, Response } from 'express';
 import { LTO } from '@ltonetwork/lto';
 import { verify } from '@ltonetwork/http-message-signatures';
+import { ConfigService } from '../config/config.service';
 
 @Injectable()
 export class VerifySignatureMiddleware implements NestMiddleware {
   private readonly lto: LTO;
 
-  constructor() {
-    this.lto = new LTO();
+  constructor(private readonly config: ConfigService) {
+    const networkID = this.config.getNetworkId();
+    this.lto = new LTO(networkID);
   }
 
   async verify(req: Request, res: Response): Promise<boolean> {
     try {
+      const fullUrl = `${req.protocol}://${req.get('host')}${req.originalUrl}`;
```

Comment:
This should be solved in https://github.com/ltonetwork/http-message-signatures/, not here